src/Pim/Component/Catalog/Updater/Remover/MultiSelectAttributeRemover.php

```diff
@@ -53,46 +52,34 @@ public function removeAttributeData(
         $this->checkLocaleAndScope($attribute, $options['locale'], $options['scope'], 'multi-select');
```

Comment:
Yes, needed because we don't directly provide the data to the product builder, we make some operations on it (removing codes), and we need to be sure we can perform those operations (that we have an array of string, for instance).